mitodl/micromasters
# Added program.price, removed CoursePrice

#### What are the relevant tickets?
Fixes #1144

#### What's this PR do?
Removes `CoursePrice` and moves its price into a new field `Program.price`. If there isn't the same price for each `CourseRun` the migration should fail, leaving the original data intact.

#### How should this be manually tested?
Run the migrations, then run them in reverse order. Checkout master and everything should still work like before. Also run `validate_db` after running migrations to confirm that that still works.
